crates/bevy_render/src/color/mod.rs

```diff
@@ -126,6 +126,15 @@ impl Color {
     pub const YELLOW_GREEN: Color = Color::rgb(0.6, 0.8, 0.2);
 
     /// New `Color` from sRGB colorspace.
+    ///
+    /// # Arguments
+    ///
+    /// * `r` - Red component. [0.0, 1.0]
+    /// * `g` - Green component. [0.0, 1.0]
+    /// * `b` - Blue component. [0.0, 1.0]
```

Comment:
```suggestion
    /// * `r` - Red component. [0.0..1.0]
    /// * `g` - Green component. [0.0..1.0]
    /// * `b` - Blue component. [0.0..1.0]
```
Does this improve the readability of the documentation? Want to avoid people taking it for an array

Reply:
Personally, I'd also remove the brackets so it would just be `0.0..1.0`

Reply:
I guess that would be `..=`. Unclear to me whether this is actually more or less confusing, personally.

Again, this style is inherited from other `Color` docs (I see an example of this style in `pbr_material.rs` as well.

Reply:
@alice-i-cecile?

Reply:
I've been using this interval notation elsewhere, and find it quite clear. The ability to readily distinguish between inclusive and exclusive ranges is especially valuable.

Reply:
So `0.0..=1.0` then?

Reply:
My preference is for the 3rd here. We should go with that for this PR, and then encode it (or a different choice) in the style guide.